doc/sphinx/src/models.rst

```diff
@@ -338,6 +338,38 @@ Note: sometimes temperatures are measured in eV for which the conversion is
 
 Sesame units are equivalent to the mm-mg-µs unit system.
 
+The ``MeanAtomicProperties`` struct
+------------------------------------
+
+Several analytic equations of state optionally accept mean atomic mass
+and number as physics properties. These are the average number of
+nucleons and protons in a constituent nucleus respectively. They are
+not necessarily integers, as a given material may be made up of
+multiple kinds of atom. For example, dry air contains both nitrogen
+and oxygen.
+
+The mean atomic mass and number are frequently carried in the
+container struct
+
+.. code-block:: cpp
+
+  struct MeanAtomicProperties {
+    Real Abar, Zbar;
+
+    // default is hydrogen
+    static constexpr Real DEFAULT_ABAR = 1.0;
+    static constexpr Real DEFAULT_ZBAR = 1.0;
```

Comment:
maybe this will become clearer as I read on... but an immediate question is `MeanAtomicMass` vs `MeanMolecularWeight`? 

When constructing an ideal gas for H2, for example, what is singularity's expectation for the extra argument? If you are setting mu:

  p = rho k T / (mu mp) 

then I wonder if `MeanMolecularWeight` makes more sense, since we want to pass mu ~= 2.  

This concern is irrelevant for monatomic gases.

Reply:
Hmm... I see your point as far as computing the value for an ideal gas from a collection of molecules. I worry a bit that there's an ambiguity here... if the ionization state is for a single atom rather than a molecule. It's possible that there are really two interpretations here.

What do you think of renaming the functions MeanUnitMass and MeanUnitCharge or something? That way it can be either, depending on context?

Reply:
ok after some discussion offline, I'm going to make this an issue #442 and leave the implementation as-is for now. I think it's a good point though. We should keep it in mind going forward. :+1: